tests/php/sync/test_class.jetpack-sync-integration.php

```diff
@@ -2,6 +2,52 @@
 
 class WP_Test_Jetpack_Sync_Integration extends WP_Test_Jetpack_Sync_Base {
 
+	function test_get_sync_status() {
+		$no_checksum = Jetpack_Sync_Actions::get_sync_status();
+		$this->assertArrayNotHasKey( 'posts_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'comments_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'post_meta_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'comment_meta_checksum', $no_checksum );
+
+		$kitchen_sink_checksum = Jetpack_Sync_Actions::get_sync_status(
```

Comment:
Also, it would be a good way to split this test to separate cases (we're currently testing several cases in the same test)

Reply:
I see nothing wrong with that.